contrib/backup/functions

```diff
@@ -191,11 +207,12 @@ function restore_zammad () {
       ${ZAMMAD_DIR}/bin/rake db:create
     fi
 
-    echo "# Restoring PostgrSQL DB"
-    zcat ${BACKUP_DIR}/${RESTORE_DB_DATE}_zammad_db.${DB_FILE_EXT}.gz | su -c "psql -d ${DB_NAME}" zammad
+    echo "# Restoring PostgreSQL DB"
+    zcat < ${BACKUP_DIR}/${RESTORE_DB_DATE}_zammad_db.${DB_FILE_EXT}.gz \
```

Comment:
The stdin redirection should not be needed:

```suggestion
    zcat ${BACKUP_DIR}/${RESTORE_DB_DATE}_zammad_db.${DB_FILE_EXT}.gz \
```

Reply:
This is needed on macOS, otherwise zcat complains about a missing `*.gz.Z` file.